invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/PatronDetails/components/ItemsSearch/components/ItemsResultsList/ItemsResultsList.js

```diff
@@ -33,33 +31,35 @@ export class ItemsResultsList extends Component {
     }, ES_DELAY);
   };
 
-  actions(item, itemState) {
+  actions = ({ row }) => {
+    // NOTE: will remove these after review
+    // itemState = row.metadata.status
```

Comment:
I am not sure I understand this comment

Reply:
I was trying to follow the variable naming, its a leftover